packages/gatsby/src/redux/types.ts

```diff
@@ -14,6 +14,7 @@ export interface IRedirect {
   toPath: string
   isPermanent?: boolean
   redirectInBrowser?: boolean
+  ignoreCase: boolean
```

Comment:
here it's fine cause it should always be set.

Reply:
Indeed and will be since there is a default